README.md

```diff
@@ -43,6 +43,8 @@ Check out our [Next.js deployment documentation](https://nextjs.org/docs/deploym
 
 - [Neka Blair](https://github.com/istoga)
 - [Ojoachele Onuh](https://github.com/Achele)
+- Chibuzo Idia (https://github.com/Domicaren1)
```

Comment:
This wouldn't come out as 
[Chibuzo Idia](https://github.com/Domicaren1)

Reply:
I don't understand @annonymousauthority

Reply:
@Domicaren1 Compare your entry with the others. What is missing from your entry? These name links are in Markdown format.

Others are: [Name](link)

Yours is: Name (link)

Do you see the difference?

Reply:
yes ma @msarit . I've seen the difference